sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultDataAction.cs

```diff
@@ -6,7 +6,32 @@
 
 namespace Azure.Security.KeyVault.Administration
 {
-    [CodeGenModel("DataAction")]
```

Comment:
Replace CodeGenModel with CodeGenType and revert the rest of the changes to this file.